build.gradle.kts

```diff
@@ -30,21 +29,10 @@ extensions.configure<ApiValidationExtension> {
 }
 
 allprojects {
-    repositories {
-        google()
-        mavenCentral()
-
-        // https://github.com/Kotlin/dokka/issues/41
-        jcenter {
```

Comment:
Dokka <1.0 has not been fully migrated to `mavenCentral` so this is needed.

Reply:
Ok